README.md

```diff
@@ -6,7 +6,62 @@
 [![codecov](https://codecov.io/gh/material-motion/motion-interchange-objc/branch/develop/graph/badge.svg)](https://codecov.io/gh/material-motion/motion-interchange-objc)
 [![CocoaPods Compatible](https://img.shields.io/cocoapods/v/MotionInterchange.svg)](https://cocoapods.org/pods/MotionInterchange)
 [![Platform](https://img.shields.io/cocoapods/p/MotionInterchange.svg)](http://cocoadocs.org/docsets/MotionInterchange)
-[![Docs](https://img.shields.io/cocoapods/metrics/doc-percent/MotionInterchange.svg)](http://cocoadocs.org/docsets/MotionInterchange)
+
+"Magic numbers" — those lonely, abandoned values without a home — are often one of the first things
+targeted in code review for cleanup. And yet, numbers related to animations may go unnoticed and
+left behind, scattered throughout a code base with little to no organizational diligence. These
+forgotten metrics form the backbone of mobile interactions and are often the ones needing the most
+care - so why are we ok leaving them scattered throughout a code base?
+
+```objc
+// Let's play "find the magic number": how many magic numbers are hidden in this code?
+[UIView animateWithDuration:0.230
+                      delay:0
+                      options:UIViewAnimationOptionCurveEaseOut
+                      animations:^{
+                        myButton.position = updatedPosition;
+                      }
+                      completion:nil];
+// Hint: the answer is not "one, the number 0.230".
+```
+
+The challenge with extracting animation magic numbers is that we often don't have a clear
+definition of *what an animation is composed of*. An animation is not simply determined by its
+duration, in the same way that a color is not simply determined by how red it is.
+
+The traits of an animation — like the red, green, and blue components of a color — include the
+following:
+
+- Delay.
+- Duration.
+- Timing curve.
+- Repetition.
+
+Within this library you will find simple data types for storing and representing animation
+traits so that the lost magic numbers for animation your code can find a place to call home.
```

Comment:
Based on the section above, It feels like the purpose of this library is to centralize "magic numbers" for animations. However, my understanding is that the whole Material Motion project is about:

1. Providing a solid foundation for motion/animation that will match Material Guidelines
2. Provide a more precise form of animation control than what is exposed by UIKit, but easier than CoreAnimation

While I agree that magic numbers are a nuisance in any code base, I think that Material Motion READMEs should highlight the two benefits above.  I feel that they provide a greater value to clients than organizing magic numbers.

Perhaps a better example might be to highlight a situation that can't be handled (easily) with UIKit, but where CoreAnimation is a bit gnarly to work with?

Reply:
> Providing a solid foundation for motion/animation that will match Material Guidelines

The Interchange and Animator solve the "Providing a solid foundation for motion/animation" part, MDC iOS is the Material Guidelines part.

> Provide a more precise form of animation control than what is exposed by UIKit, but easier than CoreAnimation.

I'll be covering this in the MotionAnimator docs next. I think that most people will use the Material Motion Animator as their primary dependency, with the interchange being a secondary dependency. As such I wanted to emphasize the importance of the interchange separately from the animator.

For example, in the animator docs I'll include feature comparison tables between UIKit, Core Animation, and MotionAnimator, in order to emphasize that MotionAnimator is a sweet spot between UIKit and Core Animation (additive, implicit animation support for all!).

Reply:
Sounds good to me. 👍